cocotb/utils.py

```diff
@@ -196,7 +202,14 @@ def unpack(ctypes_obj, string, bytes=None):
         :exc:`ValueError`: If length of *string* and size of *ctypes_obj*
             are not equal.
         :exc:`MemoryError`: If *bytes* is longer than size of *ctypes_obj*.
+
+    .. deprecated:: 1.5
+        Converting bytes to a ctypes object can be done with :meth:`from_buffer_copy <ctypes._CData.from_buffer_copy>`.
```

Comment:
Mobile won't let me suggest it, but ``:meth:`from_buffer_copy <ctypes._CData.from_buffer_copy>``` can be spelt ``:meth:`~ctypes._CData.from_buffer_copy```